owncloudApp/src/main/res/values-it/strings.xml

```diff
@@ -266,8 +266,7 @@
   <string name="auth_wtf_reenter_URL">Stato inatteso; digita nuovamente l\'indirizzo del server</string>
   <string name="auth_expired_oauth_token_toast">Autorizzazione scaduta. Richiedi l\'autorizzazione</string>
   <string name="auth_expired_basic_auth_toast">Digita la password attuale</string>
-  <string name="auth_expired_saml_sso_token_toast">Sessione scaduta. Riconnettiti</string>
-  <string name="auth_connecting_auth_server">Connessione al server di autenticazione in corso...</string>
+    <string name="auth_connecting_auth_server">Connessione al server di autenticazione in corso...</string>
```

Comment:
Why is this line indented?

Reply:
Reason in https://github.com/owncloud/android/pull/2647#discussion_r319023746